conf/modules/actuators_uavcan.xml

```diff
@@ -15,6 +15,7 @@
   </header>
   <makefile target="ap">
     <file name="actuators_uavcan.c" dir="subsystems/actuators"/>
+    <file name="pprz_random.c" dir="math"/>
```

Comment:
with https://github.com/paparazzi/paparazzi/pull/2721 it should be specified as an explicit dependency to the math module